types/es-abstract/5/CheckObjectCoercible.d.ts

```diff
@@ -0,0 +1,2 @@
+declare function CheckObjectCoercible<T>(value: T, errorMessage?: string): NonNullable<T>;
```

Comment:
```suggestion
declare function CheckObjectCoercible<T extends {}>(value: T, errorMessage?: string): NonNullable<T>;
```

Reply:
This returns `NonNullable<T>`, which correctly describes what this function does.

Reply:
so if i call `CheckObjectCoercible<null>`, TS will correctly error out because null isn't non-nullable?

Reply:
Yep, `Type 'null' does not satisfy the constraint '{}'.`

Reply:
@sandersn wait, *almost*? how is `unknown` different from `{} | null | undefined`??